zerver/lib/import_realm.py

```diff
@@ -961,49 +961,36 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
     # import the supporting data structures, which may take a bit.
     realm_properties = dict(**data["zerver_realm"][0])
     realm_properties["deactivated"] = True
-    realm = Realm(**realm_properties)
 
-    if realm.notifications_stream_id is not None:
-        notifications_stream_id: Optional[int] = int(realm.notifications_stream_id)
-    else:
-        notifications_stream_id = None
-    realm.notifications_stream_id = None
-    if realm.signup_notifications_stream_id is not None:
-        signup_notifications_stream_id: Optional[int] = int(realm.signup_notifications_stream_id)
-    else:
-        signup_notifications_stream_id = None
-    realm.signup_notifications_stream_id = None
```

Comment:
Can you clarify, in the commit mesasge, what made this block unnecessary?

Reply:
Thanks for the suggestion. This seems better. Updated the commit message.